From f740469bb064b9568ac8485a2270c0a4b3805aa0 Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Thu, 24 Jan 2013 12:37:44 +0800
Subject: [PATCH 2/5] ENGR00133115 Make .mov, .mkv, .webm .flv, .asf, .wmv to
 streaming

            Add quiktime, matroska, flv, asf, wmv mimetype for http streaming,
            to make .mov, .mkv, .webm, .flv, .asf, .wmv streams to streaming not
            download source.

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 AndroidManifest.xml | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index dc43f76..e187061 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -71,6 +71,13 @@
                 <data android:mimeType="video/webm" />
                 <data android:mimeType="video/avi" />
                 <data android:mimeType="application/sdp" />
+                <data android:mimeType="video/x-flv" />
+                <data android:mimeType="video/x-msvideo" />
+                <data android:mimeType="video/x-ms-asf" />
+                <data android:mimeType="video/x-ms-wmv" />
+                <data android:mimeType="video/mpeg" />
+                <data android:mimeType="video/quicktime" />
+                <data android:mimeType="video/x-matroska" />
              </intent-filter>
              <intent-filter>
                 <!-- HTTP live support -->
@@ -159,6 +166,13 @@
                 <data android:mimeType="video/3gpp" />
                 <data android:mimeType="video/3gpp2" />
                 <data android:mimeType="application/sdp" />
+                <data android:mimeType="video/x-flv" />
+                <data android:mimeType="video/x-msvideo" />
+                <data android:mimeType="video/x-ms-asf" />
+                <data android:mimeType="video/x-ms-wmv" />
+                <data android:mimeType="video/mpeg" />
+                <data android:mimeType="video/quicktime" />
+                <data android:mimeType="video/x-matroska" />
             </intent-filter>
         </activity>
 
-- 
1.8.0

